sdks/java/io/mqtt/src/test/java/org/apache/beam/sdk/io/mqtt/MqttIOTest.java

```diff
@@ -78,7 +78,7 @@ public void startBroker() throws Exception {
   }
 
   @Test(timeout = 60 * 1000)
-  @Category(RunnableOnService.class)
+  @Category(NeedsRunner.class)
```

Comment:
And also this is done in all the other IOs, so I am trying to get things to be similar..